src/d/alexa-openwebif/source/intents/zap.d

```diff
@@ -26,10 +27,20 @@ final class IntentZapTo : BaseIntent
 		auto targetChannel = event.request.intent.slots["targetChannel"].value;
 		Subservice matchedServices;
 		auto switchedTo = getText(TextId.ZapFailedSSML);
+		AlexaResult result;
```

Comment:
pulling up definition of result is also not necessary in any case when not uselessly setting result with the return of `returnError`